src/components/vote/Vote.js

```diff
@@ -1,16 +1,14 @@
 import React, { Component } from 'react';
 import VoteList from './VoteList';
-import CreateCategory from './CreateCategory';
-import DropdownExampleSelection from './Dropdown';
+import CreateAward from './CreateAward';
 
 /* eslint-disable */
 class Vote extends Component {
   render() {
     return (
       <>
-        <CreateCategory />
+        <CreateAward />
```

Comment:
When you get a chance, remove the `eslint-disable` comment and convert this to a functional component